modelbaker/dataobjects/layers.py

```diff
@@ -341,6 +341,20 @@ def real_id(self):
         else:
             return None
 
+    @property
+    def oid_domain(self):
+        t_ili_tid_field = self.t_ili_tid_field
```

Comment:
```suggestion
        t_ili_tid_field = self.t_ili_tid_field()
```

Missing parentheses?

Reply:
Ah no its a property...